template.yml.erb

```diff
@@ -225,12 +368,21 @@ Resources:
       FunctionName: !Ref StartSessionAndRelayMessagesFunction
       Principal: apigateway.amazonaws.com
 
-  WebsocketAuthorizerLambda:
+# AWS does not seem to be case sensitive, but is case aware. Therefore, we can't update from
+# Websocket here to WebSocket as is used elsewhere when updating an existing stack. Updating this is
+# a step we should take when we create our non-beta environment from scratch. At that point, we
+# should also update uses of HttpAPI and WebSocketAPI to be HttpApi and WebSocketApi, respectively,
+# to match the AWS standard elsewhere in this template. Example: "AWS::ApiGatewayV2::ApiMapping"
+<%{
+  Http: {LambdaName: "Http", Handler: "http_authorizer_function", Description: "'Authorize PUT by decoding JWT in Authorization querystring parameter.'"},
```

Comment:
Nit: can this be broken up onto multiple lines for readability?

Reply:
👍